pkg/client/v1/model/volume.go

```diff
@@ -4,182 +4,478 @@
-	NumConnections float64 `json:"num_connections,omitempty"`
-	// NumIscsiConnections
-	NumIscsiConnections float64 `json:"num_iscsi_connections,omitempty"`
-	// NumFcConnections
-	NumFcConnections float64 `json:"num_fc_connections,omitempty"`
-	// InheritAcl
-	InheritAcl bool `json:"inherit_acl,omitempty"`
-	// AppUuID
-	AppUuID string `json:"app_uuid,omitempty"`
-	// FolderID
-	FolderID string `json:"folder_id,omitempty"`
-	// FolderName
-	FolderName string `json:"folder_name,omitempty"`
-	// CachingEnabled
-	CachingEnabled bool `json:"caching_enabled,omitempty"`
-	// PreviouslyDeduped
-	PreviouslyDeduped bool `json:"previously_deduped,omitempty"`
-	// DedupeEnabled
-	DedupeEnabled bool `json:"dedupe_enabled,omitempty"`
-	// VpdT10
-	VpdT10 string `json:"vpd_t10,omitempty"`
-	// VpdIeee0
-	VpdIeee0 string `json:"vpd_ieee0,omitempty"`
-	// VpdIeee1
-	VpdIeee1 string `json:"vpd_ieee1,omitempty"`
-	// AppCategory
-	AppCategory string `json:"app_category,omitempty"`
-	// LimitIops
-	LimitIops int32 `json:"limit_iops,omitempty"`
-	// LimitMbps
-	LimitMbps int32 `json:"limit_mbps,omitempty"`
-	// NeedsContentRepl
-	NeedsContentRepl bool `json:"needs_content_repl,omitempty"`
-	// ContentReplErrorsFound
-	ContentReplErrorsFound bool `json:"content_repl_errors_found,omitempty"`
-	// LastContentSnapBrCgUID
-	LastContentSnapBrCgUID float64 `json:"last_content_snap_br_cg_uid,omitempty"`
-	// LastContentSnapBrGID
-	LastContentSnapBrGID float64 `json:"last_content_snap_br_gid,omitempty"`
-	// LastContentSnapID
-	LastContentSnapID float64 `json:"last_content_snap_id,omitempty"`
-	// CksumLastVerified
-	CksumLastVerified float64 `json:"cksum_last_verified,omitempty"`
-	// PreFilter
-	PreFilter string `json:"pre_filter,omitempty"`
-	// SrepLastSync
-	SrepLastSync float64 `json:"srep_last_sync,omitempty"`
-	// SrepResyncPercent
-	SrepResyncPercent float64 `json:"srep_resync_percent,omitempty"`
+   
+    // Identifier for the volume.
+    
+ 	ID *string `json:"id,omitempty"`
+   
+    // Name of the volume.
+    
+ 	Name *string `json:"name,omitempty"`
+   
+    // Fully qualified name of volume.
+    
+ 	FullName *string `json:"full_name,omitempty"`
+   
+    // Name of volume used for object search.
+    
+ 	SearchName *string `json:"search_name,omitempty"`
+   
+    // Volume size in mebibytes. Size is required for creating a volume but not for cloning an existing volume.
+    
+   	Size *int64 `json:"size,omitempty"`
+   
+    // Text description of volume.
+    
+ 	Description *string `json:"description,omitempty"`
+   
+    // Name of performance policy.
+    
+ 	PerfpolicyName *string `json:"perfpolicy_name,omitempty"`
+   
+    // Identifier of the performance policy. After creating a volume, performance policy for the volume can only be changed to another performance policy with same block size.
+    
+ 	PerfpolicyID *string `json:"perfpolicy_id,omitempty"`
+   
+    // Amount of space to reserve for this volume as a percentage of volume size.
+    
```

Comment:
Let's drop these extra lines between comment and name.

Reply:
Sure, will fix it.